### ISSUE TITLE
[Improvement] Adding a summary of cluster infrastructure in the templates list

### ISSUE THREAD

**Issue by varodrig** [stolostron/cluster-templates-operator#184]
Adding a summary such as : 2 workers nodes - aws. In the summary template could help decide which one to use instead of click on all the templates. If companies start using this they might have hundreds of templates and it's difficult to remember which one has what. They could also use this infrastructure label to order or filter templates . Potential attributes: infrastructure, cluster size. 

![image](https://github.com/stolostron/cluster-templates-operator/assets/64439402/81087bee-79da-466b-9dff-eb45abd42266)

**Comment by batzionb** [comment]
Perhaps we can enable quick viewing of the cluster template description from the table.

**Comment by varodrig** [comment]
yes, that will be great! will that help to filter/order as well?
